apollo-entity/README.md

```diff
@@ -0,0 +1,134 @@
+Apollo Entity
+=============
+
+A set of Apollo Middleware for working with entity types in route handlers.
+
+## Why
+
+A common pattern when writing route handlers is to serialize and deserialize to some API level
+entity types. Writing such (de)serialization code quickly becomes repetitive and always looks
+the same.
+
+Ideally, one would like to have endpoint handlers with straight forward signatures like
+
+```java
+Person updatePerson(String id, Person person) {
+  // ...
+}
+```
+
+With a route defined as:
+
+```java
+Route.sync(
+    "PUT", "/person/<id>",
+    rc -> person -> updatePerson(rc.pathArgs().get("id"), person))
+```
+
+However, this does not work. Theres nothing handling how the `Person` type is being read to and
+from a `ByteString` which is the payload type Apollo works with.
+
+At this point, you'll start writing some custom middlewares using something like Jackson. This
+library contains exactly those middlewares, ready to use directly with your routes.
+
+
+## Getting started
+
+Add Maven dependency:
+
+```xml
+<dependency>
+    <groupId>com.spotify</groupId>
+    <artifactId>apollo-entity</artifactId>
+    <version>1.0.5</version>
+</dependency>
+```
+
+_See the [project skeleton] docs for how to use a Maven BOM for dependency versions_
```

Comment:
But it's linked to https://github.com/spotify/apollo/tree/master/apollo-http-service#maven which is added in this PR.